docker-compose.yml

```diff
@@ -42,35 +42,72 @@ services:
     command: [ "python", "main.py", "--env", $ENV ]
   test:
     container_name: full_text_searcher_test
-    build: .
+    build:
+      context: .
+      target: runtime
+      dockerfile: Dockerfile
     volumes:
       - .:/app
     command: [ "pytest" ]
     depends_on:
       solr-lss-dev:
         condition: service_healthy
   solr-lss-dev:
-    image: ghcr.io/hathitrust/full-text-search-embedded_zoo_data_sampling:example-8.11
+    image: ghcr.io/hathitrust/full-text-search-cloud:shards-docker
     container_name: solr-lss-dev
     ports:
-        - "8983:8983"
+      - "8983:8983"
+    environment:
+      ZK_HOST: zoo1:2181
+      SOLR_OPTS: -XX:-UseLargePages
+    env_file:
+      - .env
```

Comment:
docker-compose references this `.env` as well as the solr user& password env vars, but the `.env` file isn't included with sample values, and I didn't see anything in the instructions for setting that up.

Reply:
.env file is generated from the .env.example that is included in the repository when the script ./init.sh is executed. I will add this information to the documentation. I have a question: Using this logic to store the variables in a .env file will require some change on our GitHub actions? This repository does not have GitHub actions yet. I will add a task for that.

Reply:
I don't think it should require any change in the github actions so long as the test action runs init.sh

Reply:
Ok, I should figure out how to use the init.sh file in the GitHub action. Probably we already do that on some repository I could use as a reference.